app/src/main/java/com/google/android/samples/socialite/ui/camera/CameraViewModel.kt

```diff
@@ -310,6 +317,12 @@ data class ViewFinderState(
     val lensFacing: Int = CameraSelector.LENS_FACING_BACK,
 )
 
+enum class ImageCaptureState {
+    PENDING,
+    IMAGE_CAPTURE_SUCCESS,
+    IMAGE_CAPTURE_FAIL,
+}
+
```

Comment:
The 'else' statement appears to merely manage the state without performing any significant action. Is there a specific reason for choosing an enum class over a sealed class?

Reply:
@Goooler 
oh... I think you're right, thanks.
I've understood that enum classes enhance readability when a restricted type hierarchy is not required by the current code